FormHoneypotPlugin.inc.php

```diff
@@ -24,6 +24,14 @@ class FormHoneypotPlugin extends GenericPlugin {
 		'formHoneypotMinimumTime' => 'int',
 		'formHoneypotMaximumTime' => 'int',
 	);
+	/**
+	 * @var $currentOjsVersion object
```

Comment:
This can be explicitly typed as a `Version`.

Reply:
Gotcha. Addressed in cbac7df.